dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -164,4 +164,7 @@ useSemiColonAsDelimiterInURI.name=Use semicolon as delimiter in URI
 useSemiColonAsDelimiterInURI.desc=Toggle to use the semicolon as a delimiter in the request URI. The default value is false. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.useSemiColonAsDelimiterInURI. 
 
 servletDestroyWaitTime.name=Wait for an active request to complete
-servletDestroyWaitTime.desc=Wait time in seconds for an active request to complete when the owning application is stopped. The default value is 60 seconds. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.servletDestroyWaitTime.
\ No newline at end of file
+servletDestroyWaitTime.desc=Wait time in seconds for an active request to complete when the owning application is stopped. The default value is 60 seconds. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.servletDestroyWaitTime.
+
+servletPathForDefaultMapping.name=Set the servlet path value for a default mapping
+servletPathForDefaultMapping.desc=Set the servlet path value to the request URI minus the context path and the path info is null when a servlet is used as a default mapping. The default value is true for the Servlet 4.0 feature or above, and false for other Servlet features. Also when mapping to the /* pattern, the servlet path will be empty and the path info will start with a leading /.
```

Comment:
Set the servlet path value to the request URI minus the context path and the path info is null when a servlet is used as a default mapping. The default value is true for the Servlet 4.0 feature or above, and false for other Servlet features. Also when mapping to the /* pattern, the servlet path will be empty and the path info will start with a leading /.